doc/adr/2023-05-03-patch-releases-are-ABI-compatible.md

```diff
@@ -0,0 +1,20 @@
+**Title**: Patch releases **must** be ABI compatible.
+
+**Status**: accepted
+
+**Context**: We create a few patch releases each year. These are always bug
+fixes. We have never introduced API changes in a patch release, nor should we.
+However, until this ADR is adopted our patch releases do change the ABI, because
+the patch number is embedded in the inline namespace
```

Comment:
This is a run-on sentence. Consider this slightly shorter run-on sentence:

"Prior to this ADR, our patch releases would change the ABI, because the patch number was embedded in the inline namespace ..."